app/assets/stylesheets/views/_taxons.scss

```diff
@@ -243,14 +242,4 @@
       font-weight: bold;
     }
   }
-
-  .subsection-content {
-    display: none;
-  }
-
-  .subsection-is-open {
-    .subsection-content {
-      display: block;
```

Comment:
Is `subsection-content` still a `block` in the new version? Is there an equivalent of this line?